src/interfaces/messages/messages/messages-get.ts

```diff
@@ -0,0 +1,53 @@
+import type { SignatureInput } from '../../../jose/jws/general/types.js';
+import type { MessagesGetDescriptor, MessagesGetMessage } from '../types.js';
+
+import { parseCid } from '../../../utils/cid.js';
+import { validateAuthorizationIntegrity } from '../../../core/auth.js';
+import { DwnInterfaceName, DwnMethodName, Message } from '../../../core/message.js';
+
+export type MessagesGetOptions = {
+  messageCids: string[];
+  authorizationSignatureInput: SignatureInput;
+};
+
+export class MessagesGet extends Message<MessagesGetMessage> {
+  public static async parse(message: MessagesGetMessage): Promise<MessagesGet> {
+    Message.validateJsonSchema(message);
+    this.validateMessageCids(message.descriptor.messageCids);
```

Comment:
NIT: i personally avoid using `this` in a `static` context as it can be confusing at a glance (i.e. only use `this` to represent an instance, not the `class`).  id suggest using `MessagesGet.validateMessageCids` instead